### ISSUE TITLE
Delete rdryad and testdat from package list?

### ISSUE THREAD

**Issue by karthik** [ropensci/roweb#190]
Neither work or on any timeline to be finished. Rather than pad the list with packages that don't exist, maybe we could remove them?

Others on the chopping block:

RMendeley (no updates planned)
ImpactStory (If someone asks we can point them to the deprecated package, but even that is useless since we don't have any API on the other side for the code to work).
rhindawi 
rMaps
elife

**Comment by karthik** [comment]
Also https://github.com/ropensci/rbefdata

**Comment by sckott** [comment]
- [ ] rdryad
- [ ] testdat
- [ ] RMendeley
- [ ] ImpactStory 
- [ ] rhindawi 
- [ ] rMaps
- [ ] elife

**Comment by sckott** [comment]
removed a bunch

**Comment by cboettig** [comment]
fantastic! 

looks like this can be closed?

**Comment by mbjones** [comment]
For incomplete/unmaintained packages, maybe you could have a separate page of packages that need a sponsor/maintainer/contributors?

**Comment by sckott** [comment]
@mbjones good idea Matt. Thoughts on this @karthik @cboettig @richfitz @jennybc
